proposal/package-importer.d.ts.md

```diff
@@ -281,12 +293,19 @@ the legacy importer logic will be invoked.
 Currently, the only available package importer is `node`, which follows Node
 resolution logic to locate Sass files.
 
+An optional `entryPointPath` path can be passed to provide a starting
+`parentURL` for the Node package resolution algorithm. If not set, the default
+value is `require.main.filename`.
+
 Defaults to undefined.
 
 ```ts
 declare module '../spec/js-api/legacy/options' {
   export interface LegacySharedOptions<sync extends 'sync' | 'async'> {
-    pkgImporter?: 'node';
+    pkgImporter?: {
+      type: 'node';
+      entryPointPath?: string;
+    };
```

Comment:
I think this should just take a `NodePackageImporter` instance.

Reply:
Updated in 64dc0a12696821062abfae2c1d33c9247b1fe1b6